documentation/examples/20-7guis/06-7guis-circles/App.svelte

```diff
@@ -82,6 +83,7 @@ radius of the selected circle.
 				if (adjusting) selected = circle;
 			}}
 			fill={circle === selected ? '#ccc' : 'white'}
+			role='button' tabindex='0'
```

Comment:
I think this is another warning that would be better to ignore for now -- adding role and tabindex doesn't actually make this more accessible.